### DISCUSSION TITLE
how can i catch web callback panics?

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [rust-windowing/winit#1975] [General]
I tried unwind hook here and there, but it does not invoked. so i need to try catch all entries into module kind of or other wasm module crash detectors?

**Comment by maroider** [comment]
Can you share the exact code it is you're having trouble with?

**Reply by dzmitry-lahoda** [reply]
ok, seems our project had panics abort, so i tried to catch in js, but catching in rust works, so no need to modify callbacks

**Reply by dzmitry-lahoda** [reply]
so can close
